README.md

```diff
@@ -1,6 +1,30 @@
-# City of Helsinki Ahti project
+# City of Helsinki Ahti project 
+## available at https://ahti.app/
 
-# This repository contains the first Proof of Concept implementation by Futurice
+## This repository contains the first Proof of Concept implementation by Futurice
+## Documentation last updated 13-01-2020
+
+## General information
+
+https://drive.google.com/open?id=1Rghr1WjKDJs0Vmbkmp1yLyF6IxmX5lGn
+
+## Design sources
+Are in Figma https://www.figma.com/file/ZKAH36THbAPVs9gNdmH8N6Dh/Ahti?node-id=0%3A1
+
+## Map
+The map we use is Mapbox:
+- minimal style: `mapbox://styles/strawshield/cjxx7z1sf04rm1dl7bjryf4xf` designed by Olli Kilpi
+
+## Learn More
+The app has been bootstrapped from Create React App.  [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+
+## The setup
+We have a Google Analytics team, "Ahti Helsinki".
```

Comment:
> We have a Google Analytics team, "Ahti Helsinki".

We need to set this up so everyone in the current team has access.

Reply:
I agree!